### ISSUE TITLE
ASX Symbols Latest 3 Months Missing?

### ISSUE THREAD

**Issue by elliotmitchum** [apilayer/marketstack#22]
I'm trying to fetch an ASX symbol:

http://api.marketstack.com/v1/eod/latest?access_key=KEY&symbols=CBA.XASX

However, the date is 3 months behind:

```
{
  "pagination": {
    "limit": 100,
    "offset": 0,
    "count": 1,
    "total": 1
  },
  "data": [
    {
      "open": 82.51,
      "high": 82.55,
      "low": 81.72,
      "close": 82.08,
      "volume": 1479288,
      "adj_high": null,
      "adj_low": null,
      "adj_close": 82.08,
      "adj_open": null,
      "adj_volume": null,
      "split_factor": 1,
      "symbol": "CBA.XASX",
      "exchange": "XASX",
      "date": "2021-02-22T00:00:00+0000"
    }
  ]
}
```

I've tried to fetch with specific dates:

http://api.marketstack.com/v1/tickers/CBA.XASX/eod/2021-05-22?access_key=KEY

But again, nothing?

It seems strange that it's exactly 3 months behind. My only guess is that it's because I'm on a free plan. I don't see any information stating that ASX tickers are 3 months behind.

**Comment by buddhawilliams** [comment]
Well, it seems ASX data stopped being added to on 22 Feb 2021. I just registered and tried
`http://api.marketstack.com/v1/tickers/fmg.xasx/eod/latest?access_key=KEY `
and get 
```
array [
'open' => 24.3
'high' => 25.05
'low' => 24.25
'close' => 24.74
'volume' => 6595564.0
'adj_high' => null
'adj_low' => null
'adj_close' => 24.74
'adj_open' => null
'adj_volume' => null
'split_factor' => 1.0
'symbol' => 'FMG.XASX'
'exchange' => 'XASX'
'date' => '2021-02-22T00:00:00+0000'
]
```
I am off to find another data provider... :(

**Comment by elliotmitchum** [comment]
I feel you. No reply on this ticket for a couple of months as well. Let us know if you find another provider :D

**Comment by buddhawilliams** [comment]
Have signed up for https://eodhistoricaldata.com/ and that seems to work but only 20 api calls a day on the free service. I'm using that for now, though their bottom tier of $79 a month is quite pricey for what I want it for. And just because the documentation is strange - to request an Australian stock it's STOCK.AU eg 
`https://eodhistoricaldata.com/api/eod/bhp.au?api_token=demo&fmt=json `
leaving the fmt off results in a csv download.
It's the 4th API I've signed up for today, and the only one providing recent ASX data.

**Comment by elliotmitchum** [comment]
![image](https://user-images.githubusercontent.com/34354813/126085325-5dfa612e-e9d0-4c8c-bae0-e70194f77af1.png)

Looks like they have an extended pricing tier that might include ASX for $30 US?

**Comment by buddhawilliams** [comment]
Yeah - even the $20/month option would probably suit what I'm using it for. 
Though while I'm setting up and coding, the free service is adequate. Caching data to test with rather than keep downloading latest. So far it has been useful.
